modules/actors/actor-ffg.js

```diff
@@ -95,6 +95,7 @@ export class ActorFFG extends Actor {
       }
     } else if (actorData.type === "vehicle") {
       this._applyVehicleModifiers(actorData);
+      this._calculateDerivedValues(actorData);
```

Comment:
This should be fine, but the `_calculateDerivedValues` function was not designed with vehicle actors in mind so we might need to keep an eye on this in case there are any vehicle-specific edge cases which cause unexpected behaviour.